src/EditorFeatures/Test2/IntelliSense/VisualBasicCompletionCommandHandlerTests.vb

```diff
@@ -387,7 +393,8 @@ End Class
             End Using
         End Function
 
-        <MemberData(NameOf(AllCompletionImplementations))> <WpfTheory, Trait(Traits.Feature, Traits.Features.Completion)>
+        <InlineData(CompletionImplementation.Legacy)> ' Up and Down keys are supported in VSSDK tests
```

Comment:
what does this mean?

Reply:
The editor team does not provide options to Roslyn to send Up and Down key to the Editor completion popup window. We have an open issue on them to support this test on their side.

---
In reply to: [222523381](https://github.com/dotnet/roslyn/pull/29016#discussion_r222523381) [](ancestors = 222523381)